src/main/java/dev/openfga/sdk/api/configuration/ClientConfiguration.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * OpenFGA
+ * A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar.
+ *
+ * The version of the OpenAPI document: 0.1
+ * Contact: community@openfga.dev
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package dev.openfga.sdk.api.configuration;
+
+import static dev.openfga.sdk.util.StringUtil.isNullOrWhitespace;
+
+import dev.openfga.sdk.errors.FgaInvalidParameterException;
+import java.time.Duration;
+
+public class ClientConfiguration extends Configuration {
+    private String storeId;
+    private String authorizationModelId;
+
+    public void assertValidStoreId() throws FgaInvalidParameterException {
+        if (isNullOrWhitespace(storeId)) {
+            throw new FgaInvalidParameterException("storeId", "ClientConfiguration");
+        }
+    }
```

Comment:
I think this is from an earlier revision.

Call from getStoreIdChecked()
Do the same for authorizationModelId